pep-8016.rst

```diff
@@ -311,8 +324,6 @@ votes cast in a core team vote.
 TODO
 ====
 
-- Maybe add compare-and-contrast with other 801x proposals?
-
 - Ask Ian or Ernest if they're willing to be the initial returns officer.
```

Comment:
> For the initial election, the returns officer will be [TBD].

FYI I chose to use the PSF Board to organize the election creating the Steering Committee. Following votes are organized by the Steering Committee.

Reply:
I feel like I responded to this once before somewhere... but the PSF Board seems like a weird choice. AFAIK they have never organized an election for anything before. Have you asked if they're willing to do this?

Reply:
> I feel like I responded to this once before somewhere...

Yes, we already had the same discussion :-)

> but the PSF Board seems like a weird choice. AFAIK they have never organized an election for anything before. Have you asked if they're willing to do this?

Oops, I wanted to ask them, but I didn't.

Reply:
Ewa Jodlowska already replied to me (as the voice of the PSF Board): "After discussing with Ernest, we can support this if/when the time comes." Great! So in practice, Ernest would organize the vote.

> the PSF Board seems like a weird choice

The PSF Board has an authority on Python. I prefer to write "the PSF Board", rather than naming one individual person. What kind of authority has a particual person on Python? (I'm fine with Ernest, it's just the principle in general :-))